pom.xml

```diff
@@ -202,20 +138,14 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>jackson2-api</artifactId>
-            <exclusions>
-                <exclusion>
-                    <groupId>com.fasterxml.jackson.core</groupId>
-                    <artifactId>jackson-core</artifactId>
-                </exclusion>
-            </exclusions>
```

Comment:
No need for this exclusion and in fact it was breaking the build when the above duplicate dependency was removed.